### PR TITLE
rename some track select stuff

### PR DESCRIPTION
**Pull request by tonyspumoni**
Renames:

  D_80123516->gPostRaceViewPort
  D_800DD394->gLevelLoadTimer
  D_801263D8->gOpacityDecayTimer
  D_80126480->gTrackSelectViewportY
  D_800E0978->gQMarkPtr
  D_801269F4->gSelectedTrackX
  D_801269F8->gSelectedTrackY
  D_800E7D04->gQuestionMark
  func_800C4164->set_kerning
  func_8008FA54->renderTrackSelect
  D_8012A7F0->gCompactKerning
  unk80126930->TrackRenderDetails
  
  struct TrackRenderDetails:
    levelName->hubName
    unk4->trackName
    xPos->xOff
    yPos->yOff
    pad->visible
    colour->opacity
    unkE->viewPort
    unkE.imageId->border

### CODE REVIEW

File: src/font.c [hunk 1]
```diff
@@ -139,11 +139,11 @@ void load_fonts(void) {
     }
     load_font(ASSET_FONTS_FUNFONT);
     load_font(ASSET_FONTS_SMALLFONT);
-    D_8012A7F0 = 0;
+    gCompactKerning = 0;
```
Comment (FazanaJ) [inline src/font.c:142]:
```suggestion
    gCompactKerning = FALSE;
```

File: src/game.c [hunk 2]
```diff
@@ -1102,7 +1102,7 @@ void ingame_logic_loop(s32 updateRate) {
     if (D_800DD398 < 0) {
         D_800DD398 = 0;
     }
-    if (D_80123516 != 0) {
+    if (gPostRaceViewPort != 0) {
```
Comment (FazanaJ) [inline src/game.c:1105]:
```suggestion
    if (gPostRaceViewPort) {
```

File: src/game.c [hunk 3]
```diff
@@ -1112,14 +1112,14 @@ void ingame_logic_loop(s32 updateRate) {
...
-    if (D_80123516) {
+    if (gPostRaceViewPort) {
         i = func_80095728(&gCurrDisplayList, &gGameCurrMatrix, &gGameCurrVertexList, updateRate); 
         switch (i) {
             case 2:
                 buttonHeldInputs |= (L_TRIG | Z_TRIG);
                 break;
             case 1:
-                D_80123516 = 0;
+                gPostRaceViewPort = 0;
```
Comment (FazanaJ) [inline src/game.c:1122]:
```suggestion
                gPostRaceViewPort = NULL;
```

File: src/game.c [hunk 4]
```diff
@@ -1089,7 +1089,7 @@ void ingame_logic_loop(s32 updateRate) {
         func_80010994(updateRate);
         if (check_if_showing_cutscene_camera() == 0 || func_8001139C()) {
             if ((buttonPressedInputs & START_BUTTON) && (func_8006C2F0() == 0) && (D_800DD390 == 0)
-                && (sRenderContext == DRAW_GAME) && (D_80123516 == 0) && (D_800DD394 == 0) && (D_800DD398 == 0)) {
+                && (sRenderContext == DRAW_GAME) && (gPostRaceViewPort == 0) && (gLevelLoadTimer == 0) && (D_800DD398 == 0)) {
```
Comment (FazanaJ) [inline src/game.c:1092]:
```suggestion
                && (sRenderContext == DRAW_GAME) && (gPostRaceViewPort == NULL) && (gLevelLoadTimer == 0) && (D_800DD398 == 0)) {
```

File: src/game.c [hunk 5]
```diff
@@ -1321,8 +1321,8 @@ void ingame_logic_loop(s32 updateRate) {
     }
     if (((buttonHeldInputs & L_TRIG) && (sRenderContext == DRAW_GAME)) || (D_801234FC != 0)) {
         gIsPaused = FALSE;
-        D_800DD394 = 0;
-        D_80123516 = 0;
+        gLevelLoadTimer = 0;
+        gPostRaceViewPort = 0;
```
Comment (FazanaJ) [inline src/game.c:1325]:
```suggestion
        gPostRaceViewPort = NULL;
```

File: src/game.c [hunk 6]
```diff
@@ -1391,7 +1391,7 @@ void ingame_logic_loop(s32 updateRate) {
         D_801234FC = 0;
     }
     if (D_801234F8 != 0) {
-        D_80123516 = 0;
+        gPostRaceViewPort = 0;
```
Comment (FazanaJ) [inline src/game.c:1394]:
```suggestion
        gPostRaceViewPort = NULL;
```

File: src/game.c [hunk 7]
```diff
@@ -1566,15 +1566,15 @@ void func_8006DCF8(s32 updateRate) {
...
         load_level_2(D_801234F4, D_80123500, D_80123504, gLevelDefaultVehicleID);
         func_8006EC48(get_save_file_index());
         return;
     }
     if ((menuLoopResult != -1) && (menuLoopResult & 0x100)) {
         func_8006CC14();
         gIsPaused = FALSE;
-        D_80123516 = 0;
+        gPostRaceViewPort = 0;
```
Comment (FazanaJ) [inline src/game.c:1577]:
```suggestion
        gPostRaceViewPort = NULL;
```

File: src/game.c [hunk 8]
```diff
@@ -1566,15 +1566,15 @@ void func_8006DCF8(s32 updateRate) {
         D_80123508 = 0x64;
         sRenderContext = DRAW_GAME;
         gIsPaused = FALSE;
-        D_80123516 = 0;
+        gPostRaceViewPort = 0;
```
Comment (FazanaJ) [inline src/game.c:1569]:
```suggestion
        gPostRaceViewPort = NULL;
```

File: src/game.c [hunk 9]
```diff
@@ -1545,7 +1545,7 @@ void func_8006DCF8(s32 updateRate) {
     s32 menuLoopResult, temp, temp2, tempResult;
 
     gIsPaused = FALSE;
-    D_80123516 = 0;
+    gPostRaceViewPort = 0;
```
Comment (FazanaJ) [inline src/game.c:1548]:
```suggestion
    gPostRaceViewPort = NULL;
```

File: src/game.c [hunk 10]
```diff
@@ -1939,61 +1939,61 @@ GLOBAL_ASM("asm/non_matchings/game/func_8006EFDC.s")
 #endif
 
 void func_8006F140(s32 arg0) {
-    if (D_800DD394 == 0) {
-        D_800DD394 = 0x28;
+    if (gLevelLoadTimer == 0) {
+        gLevelLoadTimer = 0x28;
```
Comment (FazanaJ) [inline src/game.c:1943]:
```suggestion
        gLevelLoadTimer = 40;
```

File: src/game.c [hunk 11]
```diff
@@ -1939,61 +1939,61 @@ GLOBAL_ASM("asm/non_matchings/game/func_8006EFDC.s")
...
         D_80123524 = 0;
         D_80123526 = 0;
         if (arg0 == 1) {
             func_800C01D8(&D_800DD41C);
         }
         if (arg0 == 3) {
-            D_800DD394 = 0x11A;
+            gLevelLoadTimer = 0x11A;
```
Comment (FazanaJ) [inline src/game.c:1950]:
```suggestion
            gLevelLoadTimer = 282;
```

File: src/game.c [hunk 12]
```diff
@@ -1939,61 +1939,61 @@ GLOBAL_ASM("asm/non_matchings/game/func_8006EFDC.s")
...
             func_800C01D8(&D_800DD424);
         }
         if (arg0 == 4) {
-            D_800DD394 = 0x168;
+            gLevelLoadTimer = 0x168;
```
Comment (FazanaJ) [inline src/game.c:1954]:
```suggestion
            gLevelLoadTimer = 360;
```

File: src/game.c [hunk 13]
```diff
@@ -1939,61 +1939,61 @@ GLOBAL_ASM("asm/non_matchings/game/func_8006EFDC.s")
...
     }
 }
 
 // Unused?
 void func_8006F20C(void) {
-    if (D_800DD394 == 0) {
+    if (gLevelLoadTimer == 0) {
         func_800C01D8(&D_800DD41C);
-        D_800DD394 = 0x28;
+        gLevelLoadTimer = 0x28;
```
Comment (FazanaJ) [inline src/game.c:1967]:
```suggestion
        gLevelLoadTimer = 40;
```

File: src/game.c [hunk 14]
```diff
@@ -1939,61 +1939,61 @@ GLOBAL_ASM("asm/non_matchings/game/func_8006EFDC.s")
...
         D_80123524 = 1;
     }
 }
 
 void func_8006F254(void) {
-    if (D_800DD394 == 0) {
+    if (gLevelLoadTimer == 0) {
         func_800C01D8(&D_800DD41C);
-        D_800DD394 = 0x28;
+        gLevelLoadTimer = 0x28;
```
Comment (FazanaJ) [inline src/game.c:1975]:
```suggestion
        gLevelLoadTimer = 40;
```

File: src/game.c [hunk 15]
```diff
@@ -1939,61 +1939,61 @@ GLOBAL_ASM("asm/non_matchings/game/func_8006EFDC.s")
...
 }
 
 void func_8006F29C(void) {
-    if (D_800DD394 == 0) {
+    if (gLevelLoadTimer == 0) {
         if ((gSettingsPtr->trophies & 0xFF) == 0xFF && !(gSettingsPtr->cutsceneFlags & CUTSCENE_LIGHTHOUSE_ROCKET) && gSettingsPtr->bosses & 1) {
             gSettingsPtr->cutsceneFlags |= CUTSCENE_LIGHTHOUSE_ROCKET;
             func_800C01D8(&D_800DD41C);
-            D_800DD394 = 0x28;
+            gLevelLoadTimer = 0x28;
```
Comment (FazanaJ) [inline src/game.c:1985]:
```suggestion
            gLevelLoadTimer = 40;
```

File: src/game.c [hunk 16]
```diff
@@ -1939,61 +1939,61 @@ GLOBAL_ASM("asm/non_matchings/game/func_8006EFDC.s")
...
     }
 }
 
 void func_8006F338(s32 arg0) {
-    if (D_800DD394 == 0) {
+    if (gLevelLoadTimer == 0) {
         D_80123525 = arg0;
         func_800C01D8(&D_800DD41C);
-        D_800DD394 = 0x28;
+        gLevelLoadTimer = 0x28;
```
Comment (FazanaJ) [inline src/game.c:1996]:
```suggestion
        gLevelLoadTimer = 40;
```

File: src/menu.c [hunk 17]
```diff
@@ -4796,11 +4796,11 @@ s32 menu_game_select_loop(s32 updateRate) {
 
     gOptionBlinkTimer = (gOptionBlinkTimer + updateRate) & 0x3F;
 
-    if (D_801263D8 != 0) {
-        D_801263D8++;
-        if (D_801263D8 >= 3) {
+    if (gOpacityDecayTimer != 0) {
```
Comment (FazanaJ) [inline src/menu.c:4799]:
```suggestion
    if (gOpacityDecayTimer) {
```

File: src/menu.c [hunk 18]
```diff
@@ -7330,7 +7330,7 @@ void menu_credits_init(void) {
     D_80126BC4 = 0;
     D_80126BCC = 0;
     D_801263E0 = 0;
-    D_801263D8 = 0x28;
+    gOpacityDecayTimer = 0x28;
```
Comment (FazanaJ) [inline src/menu.c:7333]:
```suggestion
    gOpacityDecayTimer = 40;
```

**Comment by tonyspumoni** [comment]
thanks for the corrections @FazanaJ

**Review by FazanaJ — APPROVED** [review]
Yes.

**Pull request merged**